apis/v1beta1/types.go

```diff
@@ -64,6 +64,34 @@ const (
 	Windows OS = "Windows"
 )
 
+// VirtualMachinePowerOpMode represents the various power operation modes when
+// when powering off or suspending a VM.
```

Comment:
```suggestion
// VirtualMachinePowerOffMode represents the various power operation modes
// when powering off or suspending a VM.
```
Op => Off typo I think (also in other places)

Reply:
Oh sorry. This seems to be expected :D

Reply:
I used the same naming as VM operator `VirtualMachinePowerOpMode`.
https://github.com/vmware-tanzu/vm-operator/blob/3a1bf614d9f15015f44d80c723d2809a6f0032a7/api/v1alpha2/virtualmachine_types.go#L119
In VM operator there are other cases for suspend and restart as well.

Reply:
Yup, all good